cli/tests/pcluster/utils/test_pcluster_utils.py

```diff
@@ -961,3 +961,42 @@ def test_get_instance_families_from_types(caplog, instance_types, error_expected
 def test_is_instance_type_format(candidate, expected_return_value):
     """Verify function that decides whether or not a string represents an instance type behaves as expected."""
     assert_that(utils.is_instance_type_format(candidate)).is_equal_to(expected_return_value)
+
+
+@pytest.mark.parametrize(
+    "snapshot_id, error_message",
+    [
+        ("snap-1234567890abcdef0", None),
+        ("snap-1234567890abcdef0", "Some error message"),
+    ],
+)
+def test_get_snapshot_info(boto3_stubber, snapshot_id, error_message):
+    """Verify that get_snapshot_info makes the expected API call."""
+    response = {
+        "Description": "This is my snapshot",
+        "Encrypted": False,
+        "VolumeId": "vol-049df61146c4d7901",
+        "State": "completed",
+        "VolumeSize": 120,
+        "StartTime": "2014-02-28T21:28:32.000Z",
+        "Progress": "100%",
+        "OwnerId": "012345678910",
+        "SnapshotId": "snap-1234567890abcdef0",
+    }
+    describe_snapshots_response = {"Snapshots": [response]}
+
+    mocked_requests = [
+        MockedBoto3Request(
+            method="describe_snapshots",
+            response=describe_snapshots_response if error_message is None else error_message,
+            expected_params={"SnapshotIds": ["snap-1234567890abcdef0"]},
+            generate_error=error_message is not None,
+        )
+    ]
+    boto3_stubber("ec2", mocked_requests)
+    if error_message is None:
+        assert_that(utils.get_ebs_snapshot_info(snapshot_id)).is_equal_to(response)
+    else:
+        with pytest.raises(SystemExit, match=error_message) as sysexit:
+            utils.get_ebs_snapshot_info(snapshot_id)
```

Comment:
Add test a test cases that sets the `raise_exceptions=True` when calling this function. You can do this by adding another parameter to go along with `snapshot_id` and `error_message`.

Reply:
Thanks! Done